tests/e2e-pw/global-setup.ts

```diff
@@ -1,27 +1,26 @@
 /**
  * External dependencies
  */
+import * as fs from 'fs';
 import { chromium, expect } from '@playwright/test';
-import fs from 'fs';
 
 /**
  * Internal dependencies
  */
-import { admin, customer } from './test-data/data/data';
+import { admin, customer } from './test-data/data';
 
 /* eslint-disable no-console */
 
 module.exports = async ( config ) => {
+	console.log( 'Global setup' );
```

Comment:
Just want to learn why we are doing the console log here as these tests will be executed by CI. Is there any other way to log the progress instead of console.log like a reports file or something?